esp/files/scripts/configmgr/configmgr.js

```diff
@@ -3312,27 +3313,30 @@ function onMenuItemClickHandleComputer(p_sType, p_aArgs, p_oValue) {
   var menuItemName = this.cfg.getProperty("text");
   var oper = "Delete";
   if (menuItemName === "New" || menuItemName === "New Range...")
-    oper = "New";
+  {
+    if (parentName === "Computers")
+    {
+      for (i = 0; i < recSet.getLength(); i++) {
+        var arr = recSet.getRecord(i).getData('domain_extra');
+        if (YAHOO.lang.isArray(arr))
+          break;
+      }
 
-  if (parentName === "Computers" && ( menuItemName === "New Range..." || menuItemName === "New")) {
-    for (i = 0; i < recSet.getLength(); i++) {
-      var arr = recSet.getRecord(i).getData('domain_extra');
-      if (YAHOO.lang.isArray(arr))
-        break;
-    }
+      if (i >= recSet.length) {
+        alert("get domain from backend");
+        return;
+      }
+
+      var type =( menuItemName === "New Range..." ) ? 0 : 3;
 
-    if (i >= recSet.length) {
-      alert("get domain from backend");
+      if (recSet.getRecord(i) != null)
+        top.document.navDT.promptNewRange(recSet.getRecord(i).getData('domain_extra'), recSet.getRecord(i).getData('computerType_extra'), type);
+      else
+        top.document.navDT.promptNewRange([], [], type);
       return;
     }
-    
-    var type =( menuItemName === "New Range..." ) ? 0 : 3;
-
-    if (recSet.getRecord(i) != null)
-      top.document.navDT.promptNewRange(recSet.getRecord(i).getData('domain_extra'), recSet.getRecord(i).getData('computerType_extra'), type);
-    else
-      top.document.navDT.promptNewRange([], [], type);
-    return;
+    else if (parentName ==="Computer Types")
```

Comment:
It might be difficult to see due to existing indentation, but this is meant to pair up w/ the " if (parentName === "Computers")" of line 3317.
However, I prob should consider the unlikely "else" case...